kafka-streams-serdes/build.gradle.kts

```
@@ -15,6 +16,7 @@ dependencies {
 
   api("org.apache.kafka:kafka-clients")
   api("org.apache.avro:avro")
+  implementation("com.google.protobuf:protobuf-java:3.23.0") // Adjust the version as needed
```

Can you please use the version from the `kafka-bom`? Also, the comment can be removed.
```suggestion
  implementation("com.google.protobuf:protobuf-java")
```
Define the version in `kafka-bom`, which might be using the version catalog.

---

U mean to use same version defined in kafka-bom right?

---

If you define the version in the bom, it is not required to specify the version here.